labgrid/remote/exporter.py

```diff
@@ -916,6 +916,13 @@ def main():
         default=None,
         help='hostname (or IP) published for accessing resources (defaults to the system hostname)'
     )
+    parser.add_argument(
+        '-f',
```

Comment:
After looking at this again, the feature seems to be required in rather special use cases, so we should drop the shortopt, only use `--fqdn` (same as `--hostname`). I would have fixed this myself and would have approved the PR, but you did not allow maintainer pushes to the PR branch.

Reply:
ok, reworked the branch to drop the -f-

Reply:
Thanks!